src/main/webapp/js/common.js

```diff
@@ -868,3 +868,56 @@ function hideSingleCollapse(e) {
     $(heading).find('a.btn').hide();
 }
 
+/**
+ * Wrapper for Bootbox.js (available at http://bootboxjs.com/)
+ * "Bootbox.js is a small JavaScript library which allows you to create programmatic dialog boxes using
+ *  Bootstrap modals"
+ */
+var BootboxWrapper = {
+
+    /**
+     * Custom alert dialog to replace default alert() function
+     * Required params: titleText and messegeText
+     */
+    showModalAlert: function(titleText, messageText, okButtonText = 'OK', color = StatusType.INFO) {
```

Comment:
`param = defaultVal` is an ES6 feature. To do it the ES5 way, pass just the param into the function and do `param || defaultValue` in places needed.